manifests/jenkins.yaml

```diff
@@ -159,15 +159,6 @@ objects:
           - name: coreos-x86-64-builder
             mountPath: /var/run/secrets/coreos-x86-64-builder
             readOnly: true
-          - name: oscontainer-secret
-            mountPath: /var/run/secrets/oscontainer-secret
-            readOnly: true
```

Comment:
I'm confused by this commit. It moves over the cosa and Fedora secrets but here we also delete the secret for the legacy location.

Hmm, are we missing a section for it in the HACKING.md to set that up?

Reply:
ehh yeah. That secret is going away in like a month so I elected to leave it out of `HACKING.md`. It already wasn't in there.

Reply:
Ack gotcha. I think ideally we'd have it there for completeness but OK as is.